### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by richbrill**
Couple of grammatical corrections :)

### CODE REVIEW

**Comment by nmstoker** [comment]
Well done! Not to detract from the excellent work on en-inflectors but the mistakes do stand out like a sore thumb given it's a language related library! 😀

File: readme.md [hunk 1]
```diff
@@ -112,23 +112,23 @@ instanceC.toPlural(); // money (no change)
...
+First of all, unless you have a dictionary of all the words and verbs that exist in English you can't really write a regular expression or an algorithm and expect to have a 100% success rate. English has been adopting words from a lot of different languages (French, Greek and Latin for example) and each one of these languages has its own rules of pluralization and singularization, let alone verb conjugation.
 
-Even with dictionaries you'll have the problem of complex and made up words like `maskedlocation`, and you might have to add dictionaries for specialties (like medicine which does actually have it's own dictionary). 
+Even with dictionaries you'll have the problem of complex and made up words like `maskedlocation`, and you might have to add dictionaries for specialities (like medicine which does actually have its own dictionary).
```
Comment (kswedberg) [inline readme.md]: "Specialties" is fine as the American variant of the word.

File: readme.md [hunk 2]
```diff
@@ -112,23 +112,23 @@ instanceC.toPlural(); // money (no change)
 
 * **Verb conjugation**
 	1. Dictionary lookup (known irregularities + 4000 common verbs)
-	2. If the passed verb is identified as infinitive then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.
+	2. If the passed verb is identified as infinitive it then applies regular expression transformations that are based on word endings, vowels and consonant phonetics.
```
Comment (kswedberg) [inline readme.md]: Should also insert a comma after "infinitive," beause the sentence starts with a dependent (subordinate) clause.

File: readme.md [hunk 3]
```diff
@@ -112,23 +112,23 @@ instanceC.toPlural(); // money (no change)
...
 	3. Tries to trim character from the beginning of the verb, thus solving prefixes (e.g. undergoes, overthrown)
 	4. Tries to stem the word and get the infinitive form, then apply regular expression transformations.
 	5. Applies regular expressions.
 
 
 ## How accurate is it?
 
-First of all, unless you have a dictionary of all the words and verbs there are in the English you can't really write a regular expression or an algorithm and expect to have a 100% success rate. English has been adopting words form a lot of different languages (French, Greek and Latin for example) and each one these languages has it's own rules of pluralization and singularization let alone verb conjugation.
+First of all, unless you have a dictionary of all the words and verbs that exist in English you can't really write a regular expression or an algorithm and expect to have a 100% success rate. English has been adopting words from a lot of different languages (French, Greek and Latin for example) and each one of these languages has its own rules of pluralization and singularization, let alone verb conjugation.
```
Comment (kswedberg) [inline readme.md]: Should insert a comma after "English" in the first sentence and before "and each one" in the second.

**Comment by alexcorvi** [comment]
Merged, thanks!

**Comment by richbrill** [comment]
No problem!

**Pull request merged**